### PR TITLE
[Fixed] Fix issue #298

### PR DESCRIPTION
**Pull request by xiliu8006**
Avoid 4 dims points as the input of points_in_boxes functions

### CODE REVIEW

File: mmdet3d/ops/roiaware_pool3d/points_in_boxes.py [hunk 1]
```diff
@@ -17,6 +17,7 @@ def points_in_boxes_gpu(points, boxes):
     """
     assert boxes.shape[0] == points.shape[0]
     assert boxes.shape[2] == 7
+    assert points.shape[2] == 3
```
Comment (ZwwWayne) [inline mmdet3d/ops/roiaware_pool3d/points_in_boxes.py:20]: , f'got unexpected shape {points.shape[2]}'

File: mmdet3d/ops/roiaware_pool3d/points_in_boxes.py [hunk 2]
```diff
@@ -83,8 +89,12 @@ def points_in_boxes_batch(points, boxes):
     Returns:
         box_idxs_of_pts (torch.Tensor): (B, M, T), default background = 0
     """
-    assert boxes.shape[0] == points.shape[0]
-    assert boxes.shape[2] == 7
+    assert boxes.shape[0] == points.shape[0], \
+        'Points and boxes should have the same batch size'
```
Comment (ZwwWayne) [inline mmdet3d/ops/roiaware_pool3d/points_in_boxes.py:93]: `f'Points and boxes should have the same batch size, got {boxes.shape[0]} and {points.shape[0]}'`

File: mmdet3d/ops/roiaware_pool3d/points_in_boxes.py (hunk 1 above)
Comment (xiliu8006) [reply mmdet3d/ops/roiaware_pool3d/points_in_boxes.py]: OK, I will modify it

File: mmdet3d/ops/roiaware_pool3d/points_in_boxes.py (hunk 2 above)
Comment (xiliu8006) [reply mmdet3d/ops/roiaware_pool3d/points_in_boxes.py]: got it, I will modify it

**Review by ZwwWayne — APPROVED** [review]

**Pull request merged**